manim/__main__.py

```diff
@@ -73,7 +73,20 @@ def main():
     else:
         config.digest_args(args)
         input_file = config.get_dir("input_file")
-        if config["use_webgl_renderer"]:
+
+        if config["use_opengl_renderer"]:
+            from manim.renderer.opengl_renderer import OpenGLRenderer
+
+            for SceneClass in scene_classes_from_file(input_file):
+                try:
+                    renderer = OpenGLRenderer()
+                    scene = SceneClass(renderer)
+                    scene.render()
+                except Exception:
+                    print("\n\n")
+                    traceback.print_exc()
+                    print("\n\n")
```

Comment:
use rich here. see how it is done previously, I changed it recently in https://github.com/ManimCommunity/manim/commit/0de873c803bc4b9ab73615d3ebf4c95f8ecba9b3